index.d.ts

```diff
@@ -60,3 +60,8 @@ console.log(await getPort({port: portNumbers(3000, 3100)}));
 ```
 */
 export function portNumbers(from: number, to: number): Iterable<number>;
+
+/**
+Clear the internal cache of locked ports.
+ */
```

Comment:
```suggestion
*/
```

Reply:
This should document the use-case for this method and potential downsides with using it.

Also need this in the readme.